### PR TITLE
Update CI for 2.1.

### PR DESCRIPTION
**Pull request by crummel**
This depends on @dseefeld's changes which are not in yet - this may need to be updated after those go in.

### CODE REVIEW

**Comment by crummel** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -69,11 +69,11 @@ def addPushJob(String project, String branch, String os, String configuration)
       def newJob = job(Utilities.getFullJobName(project, shortJobName, isPR)){
         steps{
             shell("cd ./source-build;git submodule update --init --recursive");
-            shell("cd ./source-build;./build.sh /p:ArchiveDownloadedPackages=true /p:Configuration=${configuration} ${loggingOptions}");
+            shell("cd ./source-build;./build.sh /p:ArchiveDownloadedPackages=true /p:RootRepo=known-good /p:Configuration=${configuration} ${loggingOptions}");
```
Comment (dseefeld) [inline netci.groovy]: I set the RootRepo in dir.props to known-good, so you shouldn't need to specify a RootRepo here or on the tarball build.

File: netci.groovy [hunk 2]
```diff
@@ -69,11 +69,11 @@ def addPushJob(String project, String branch, String os, String configuration)
...
             shell("cd ./source-build;./build-source-tarball.sh ../tarball-output --skip-build");
 
             // For now, perform offline build up to corefx until we work through issues with other repos
```
Comment (dseefeld) [inline netci.groovy]: This comment can go

File: netci.groovy (hunk 1 above)
Comment (crummel) [reply netci.groovy]: Removed.

File: netci.groovy (hunk 2 above)
Comment (crummel) [reply netci.groovy]: Removed.

**Comment by crummel** [comment]
@dotnet-bot test ci please

**Pull request merged**